xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.35.xml

```diff
@@ -0,0 +1,126 @@
+                        <layout columns="1" writtenLines="1">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#impressed.inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p>The stele depicts two men in low relief shaking hands.</p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history>
+                  <summary>
+                     <rs/>  <!-- content can be "fake" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date notBefore="-0400" notAfter="-0350">early fourth century BCE</date>
+                     <placeName ref="europe.greece.attica">Greece, Attica</placeName>
+                  </origin>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CAT_2"/>
+                  <biblScope unit="item">226</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#Rudolph"/>
+                  <biblScope unit="item">30-31, no. 11</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#Rudolph"/>
+                  <biblScope unit="item">34-35, no.13</biblScope>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include  href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="CDS">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="EM">More tweaks for bib only inscriptions</change>
+         <change when="2020-02-12" who="SPapandonatou">added metedata and transcription</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+         <ab>
+            <lb n="1"/><persName><name key="Lysikleides Λυσικλειδες"><gap reason="lost" quantity="3" unit="character"/>ΙΚΛΕΙΔΗΣ</name><name key="Lysistratos Λυσιστρατος"><gap reason="lost" quantity="3" unit="character"/>ΣΙΣΤΡΑΤΟ</name></persName> Α <!-- [demoticum]??? -->
```

Comment:
If you're going to lemmatize this as Lysikleides, you should use `<supplied>` for the missing characters of the name rather than giving the text as lost; otherwise, I'm not sure you should claim that as the name in a lemma (and you can leave the `<gap>`).